L1Trigger/Phase2L1ParticleFlow/test/make_l1ct_patternFiles_cfg.py

```diff
@@ -0,0 +1,98 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process("RESP", eras.Phase2C9)
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load("SimGeneral.HepPDTESSource.pythiapdt_cfi")
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+process.options   = cms.untracked.PSet( wantSummary = cms.untracked.bool(True), allowUnscheduled = cms.untracked.bool(False) )
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(1000))
+process.MessageLogger.cerr.FwkReport.reportEvery = 1
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:inputs110X.root'),
+    inputCommands = cms.untracked.vstring("keep *", 
+            "drop l1tPFClusters_*_*_*",
```

Comment:
Is this correct, not *_l1tPFClusters_*_* ? I assume `l1tPFClusters` is a module name, not type.

Reply:
it's a type name: https://github.com/cms-sw/cmssw/blob/master/DataFormats/L1TParticleFlow/interface/PFCluster.h